web/apps/serve.go

```diff
@@ -75,7 +75,7 @@ func handleIntent(c echo.Context, i *instance.Instance, slug, intentID string) {
 	if !allowed {
 		return
 	}
-	from := i.SubDomain(intent.Client).Host
+	from := i.SubDomain(intent.Client).String()
```

Comment:
Apparently somthing is done here to split the Client value, but then the previous value is re-affected https://github.com/cozy/cozy-stack/blob/master/web/intents/intents.go#L54